packages/wp-story-editor/src/components/documentPane/publish/publish.js

```diff
@@ -414,6 +447,41 @@ function PublishPanel({ nameOverride }) {
             </LabelWrapper>
           </DropdownWrapper>
         </HighlightRow>
+        {improvedAutosaves && (
+          <RevisionsWrapper>
+            <LabelWrapper>
+              <Label>
+                <LabelIconWrapper>
+                  <Icons.History width={24} height={24} aria-hidden />
+                </LabelIconWrapper>
+                <RevisionsLabel>
+                  {sprintf(
+                    /* translators: %d: number of revisions. */
+                    _n(
+                      '%d Revision',
+                      '%d Revisions',
+                      revisionCount,
+                      'web-stories'
+                    ),
+                    sprintf('%s', revisionCount)
+                  )}
+                </RevisionsLabel>
+              </Label>
+              {revisionLink && revisionId ? (
+                <Row>
+                  <Link
+                    rel="noopener noreferrer"
+                    target="_blank"
+                    href={addQueryArgs(revisionLink, { revision: revisionId })}
```

Comment:
We may want to have a help function to get a revision link from config. Makin this a little more compatible with other cmses. Thoughts @swissspidy

Reply:
The link cannot come from the config because it‘s dynamic. It should come from the REST API, which is why I‘ve iriginally suggested adding a new REST field to the API, so that we don‘t have to create the link here ourselves.

Reply:
Why can't it come from a config? How about a function like this.

```getRevisionLink( revisionId )```

Reply:
The final revision link is `revision.php?revision=<id>` where `<id>` changes every time you save the story in the editor.

Anyway, the current way is OK for now since this is in the `wp-story-editor` package, so other CMSes are not relevant here.